camera_calibration/src/camera_calibration/calibrator.py

```diff
@@ -938,6 +939,33 @@ def pt2line(x0, y0, x1, y1, x2, y2):
         else:
             return None
 
+    def update_mask(self, img, screen, radius = 35):
+        """
+        Update or if None create a mask which shows where no corner detection has occurred
+        with the same size of screen and add on top of it detection areas displayed through
+        circles which represents instead areas of the image where corners have been detected
+
+        :param img: source image
+        :type img: :class:`cvMat`
+
+        :param screen: display
+        :type screen: :class:`cvMat`
+
+        :param radius: radius of the circle which shows where corner detection has occurred
+        :type radius: :class:`int`
+        """
+        height, width = img.shape[0], img.shape[1]
+        shape = screen.shape
+        scale_height, scale_width = height / shape[0], width / shape[1]
+        # Create the binary mask
+        self.mask = numpy.full((shape[0], shape[1], 1), 255, dtype=numpy.uint8)
```

Comment:
Since detected corners never turn back to not detected, there is no need to create the mask every time you want to update it. Just create it here only when it is None... Please correct me if I am overlooking something...